api/routes/transactions.js

```diff
@@ -3,23 +3,34 @@
  */
 
 const express = require('express');
-const Sequelize = require('sequelize');
 const router = express.Router();
 
+const logger = require('../logger');
 const Transaction = require('../models/transaction');
 
 router.get('/', async (req, res, next) => {
-    const transactions = await Transaction.findAll();
-    res.send(transactions);
+    try {
+        const transactions = await Transaction.findAll();
+        res.json(transactions);
+    } catch(err) {
+        res.status(500).send(err.message);
+        logger.error('Error transaction get', err);
+    }
 });
+
 router.post('/', async (req, res, next) => {
-    let transactions = req.body,
-        upsertPromises = [];
-    transactions.forEach(transaction => {
-        upsertPromises.push(Transaction.upsert(transaction));
-    });
-    let result = await upsertPromises;
-    res.send('Transactions are successfully inserted!');
+    try {
+        let transactions = req.body,
+            upsertPromises = [];
+        transactions.forEach(transaction => {
+            upsertPromises.push(Transaction.upsert(transaction));
+        });
+        let result = await Promise.all(upsertPromises);
+        res.send('Transactions are successfully inserted!');
+    } catch(err) {
+        res.status(500).send(err.message);
+        logger.error('Error transaction post', err);
```

Comment:
please change to human readable sentence, like "An error occurred when inserting transactions."